flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionGraph.java

```diff
@@ -893,13 +1086,11 @@ public Object call() throws Exception {
 								}
 							}, executionContext);
 							break;
-						}
-						else if (numberOfRetriesLeft <= 0 && transitionState(current, JobStatus.FAILED, failureCause)) {
+						} else if (numberOfRetriesLeft <= 0 && transitionState(current, JobStatus.FAILED, failureCause)) {
 							postRunCleanup();
 							break;
 						}
-					}
-					else {
+					} else {
```

Comment:
You're touching a lot of code here (and above) for no other reason than reformatting. We have no style guide yet that enforces this formatting. In general, we try not to touch existing lines to keep the latest history accurate.